### PR TITLE
Add golang client

### PR DESCRIPTION
**Pull request by saschagrunert**
This adds an example golang client by doing the same like the rust
client.

### CODE REVIEW

File: cmd/client/main.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	if err != nil {
+		return err
+	}
+
+	version, err := response.Version()
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Version resposne: %s\n", version)
```
Comment (rphillips) [inline cmd/client/main.go:51]: typo: response

Comment (saschagrunert) [reply cmd/client/main.go]: Fixed :+1:

**Comment by haircommander** [comment]
hold this for now, I think we may want to make a fully fledged library instead of a client example

**Comment by saschagrunert** [comment]
Converted back to draft, @haircommander feel free to close if you wanna move over the code into your WIP.

**Comment by saschagrunert** [comment]
> BTW I'm interested in your experience with cap'n proto for IPC between Rust and Go

Yeah I was thinking if IPC would be usable for us as well.

**Comment by saschagrunert** [comment]
Part of https://github.com/containers/conmon-rs/pull/37

**Pull request closed without merging**